conformance/results/pyright/qualifiers_final_decorator.toml

```diff
@@ -18,3 +18,4 @@ qualifiers_final_decorator.py:126:5 - error: Function "func1" cannot be marked @
 conformance_automated = "Pass"
 errors_diff = """
 """
+ignore_errors = ["reportMissingModuleSource"]
```

Comment:
Pyright (and several others) don't actually pick up the `_qualifiers_final_decorator.pyi` stub file. Leaving that for another day.

Reply:
I'm not sure what you mean by "don't actually pick up the stub file". Can you elaborate? I'm seeing pyright resolve the import with the stub file when I run the test. Are you seeing something different?

Reply:
I think I misinterpreted the pyright error; I thought it meant pyright couldn't find the file at all, but it seems like it means only that pyright couldn't find the `.py` file, but it picked up the `.pyi` correctly.

It does appear that pytype can't resolve the import (`File "qualifiers_final_decorator.py", line 8, in <module>: Couldn't import pyi for '_qualifiers_final_decorator' [pyi-error]`), but all other type checkers are fine. Pytype also fails this test for other reasons, so we don't need to worry about that yet.